src/main/java/uk/nhs/tis/sync/dto/TrustDmsDto.java

```diff
@@ -1,20 +0,0 @@
-package uk.nhs.tis.sync.dto;
-
-import com.fasterxml.jackson.annotation.JsonInclude;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
-@NoArgsConstructor
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@Data
-public class TrustDmsDto {
```

Comment:
As per mapper, the source has a nested DTO. There are 779 "CURRENT" records with a value.

Reply:
As with SiteDTO, the version of reference-api used doesn't include that nested DTO.